src/lib/containers/StorybookComponentWrapper.tsx

```diff
@@ -22,10 +25,16 @@ export async function sessionChangeHandler() {
     profile = await getUserProfile(accessToken)
   } catch (err) {
     if (err instanceof SynapseClientError && err.status === 401) {
+      console.error(
+        'Encountered error fetching profile: ',
+        err,
+        'Signing out...',
+      )
       await signOut(() => {})
       accessToken = undefined
     }
-    console.error('Encountered error fetching profile: ', err, 'Signing out...')
```

Comment:
Move the error message inside, otherwise it was inaccurate (we didn't necessarily sign out)